firefox-ios/Client/metrics.yaml

```diff
@@ -5102,6 +5102,26 @@ fx_suggest:
     expires: never
     send_in_pings:
       - fx-suggest
+  is_clicked:
+    type: boolean
+    description: >
+      If `ping_type` is "fxsuggest-impression", indicates whether this
+      impression is for a clicked suggestion.
+      If `ping_type` is "fxsuggest-click", always `true`.
+    bugs:
+      - TODO
+    data_reviews:
+      - TODO
```

Comment:
Leaving a note to myself to file a data review request.

Reply:
Should this receive a data review before being merged?

Reply:
Yep—I'll fill out the form today; I just wanted to get the code up for review first.